stdlib/2and3/zipfile.pyi

```diff
@@ -131,12 +131,36 @@ class ZipInfo:
     compress_size: int
     file_size: int
     def __init__(self, filename: Optional[Text] = ..., date_time: Optional[_DT] = ...) -> None: ...
-    if sys.version_info >= (3, 6):
-        def is_dir(self) -> bool: ...
+    if sys.version_info >= (3, 8):
+        @classmethod
+        def from_file(cls, filename: _Path, arcname: Optional[_Path] = ..., *, strict_timestamps: bool = ...) -> ZipInfo: ...
+    elif sys.version_info >= (3, 6):
         @classmethod
         def from_file(cls, filename: _Path, arcname: Optional[_Path] = ...) -> ZipInfo: ...
+    if sys.version_info >= (3, 6):
+        def is_dir(self) -> bool: ...
     def FileHeader(self, zip64: Optional[bool] = ...) -> bytes: ...
 
+if sys.version_info >= (3, 8):
+    class Path:
+        @property
+        def name(self) -> str: ...
+        def __init__(self, root: Union[ZipFile, _Path, IO[bytes]], at: str = ...) -> None: ...
+        def open(self, mode: Text = ..., pwd: Optional[bytes] = ...) -> IO[bytes]: ...
```

Comment:
This has a `force_zip64` keyword only argument (which is missing from `ZipFile.open()` as well).
Also a minor nit: Since this is in the Python 3 branch, `Text` should just be `str`.